src/_press/cal-itp-gtfs-schedule-validator.md

```diff
@@ -4,4 +4,5 @@ title: New web tool makes GTFS validation easier for transit agency producers an
 external: https://mobilitydata.org/new-web-based-version-of-gtfs-schedule-validator-released/
 tags:
   - GTFS
+outlet: "Cal-ITP"
```

Comment:
@mrose914 In the [sheet](https://docs.google.com/spreadsheets/d/12zdDDRArtyXmm1wZJk4-vq1KvF8aZ9BWrIBIM4T7R3w/edit#gid=1455215505), this press item is in the `Cal-ITP Press Release` tab. So I added `Cal-ITP` as the name of the outlet. But the link goes to `MobilityData.org` and the dateline is set to Montreal. Is this correct or should the outlet say `MobilityData.org`?

Reply:
```suggestion
outlet: "MobilityData.org"
```